Sources/Functions/Block.swift

```diff
@@ -23,14 +23,11 @@
 /// Closure with custom arguments and return value.
 public typealias Block<Input, Output> = (Input) -> Output
 
-/// Closure with no arguments and custom return value.
-public typealias ResultBlock<Output> = Block<Void, Output>
```

Comment:
why removed?

Reply:
because it doesn't work properly.
translates to this kind of code `... = (()) -> Output`

Reply:
got the problem.
solution will be:
```swift
public typealias ResultClosure<Output> = () -> Output
```